docs/tutorials/ifci_px4_flight_controller.rst

```diff
@@ -0,0 +1,276 @@
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/module_parameters.png
+    :align: center
+    :scale: 50
+    :alt: PX4 Module Parameters
+
+    PX4 Module Parameters
+
+Once these are all set, set the ``VERTIQ_NUM_CVS`` parameter to the highest CVI that you used plus 1. This refers to the number of Control Values the flight controller should send. If you only have 4 control values, but the highest one is 11, the flight controller needs to send 12 control values, even if some intermediate ones are unused. It better to use every control value you are sending to reduce overhead on the serial line. If you are doing a quadcopter and used ``THROTTLE_CVI`` 0, 1, 2, 3, you should set the ``VERTIQ_NUM_CVS`` to 4.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/num_cvs.png
+    :align: center
+    :scale: 50
+    :alt: Num CVs
+
+    Setting the Number of Control Values
+
+The final values to set are ``TELEM_IDS_1`` and ``TELEM_IDS_2``. These control which module IDs have telemetry requested. When you click on one of these parameters, a list of checkboxes should show up. Tick each module ID that is on your bus to request telemetry from that module.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/telem_ids.png
+    :align: center
+    :scale: 50
+    :alt: Telem Modules
+
+    Setting the Modules to Request Telemetry From
+
+Actuator Setup
+==============
+
+The next step is to set up the actuator associations. Go to the actuator tab in QGroundControl settings and select the 'Vertiq IO' tab on the top right. Initially every ESC function will be disabled. Each ESC number in this section corresponds to a CVI number plus 1. For example ESC 1 corresponds to CVI 0 and so on. Based on the geometry drawing, match the motor with the appropriate ESC # (and therefore CVI). As you do, also ensure that the direciton checkbox is ticked properly. Module ID is irrelevant here.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/geometry_setup.png
+    :align: center
+    :scale: 50
+    :alt: geometry setup
+
+    Matching Motor Geometry to ESCs
+
+In the example image above, the from right module has a ``THROTTLE_CVI`` of 0, the back left module; 1, front left; 2, back left; 3. This is conveniently in order, so an additional example is shown below where this is not the case.
+
+.. figure:: ../_static/tutorial_images/ifci_px4_flight_controller/silly_geometry_setup.png
+    :align: center
+    :scale: 50
+    :alt: geometry setup
+
+    Matching Motor Geometry to ESCs with a 'Non-Conventional' Setup
+
+In the setup above the ``THROTTLE_CVI`` for the front right; 0, back right; 1, front left; 2, back left; 3. As long as the module CVI to Motor number association is correct, the right motor will spin.
+
+Spin Testing
+============
+
```

Comment:
you should have some mention of arming stuff here.

Reply:
this again.

Reply:
I'm not sure I follow. Will follow up

Reply:
Ah. Well, if someone doesn't have their module set up to arm on throttle, it's not going to spin with the actuator test. Or, if they have their arming region in a weird place, it'll be hard to arm. And what if they're set in 3d mode (I know we're not in this example), but it's something we should probably cover. Just the sentence, "This effectively arms the modules" is not necessarily true. It enables the throttle output, but all of the modules are still subject to arming regions/direction/mode etc.